### ISSUE TITLE
Is there a way to get csv (or table) files without going to the viewer

### ISSUE THREAD

**Issue by pstrope** [blobtoolkit/blobtoolkit#37]
Hi, How can I get a table of results without having to open the viewer. I ran the blobtoolkit2 and have results, all of them are .json files. I was wondering if there is a quick command line way to see the csv or any table of results. Thank you!

**Comment by rjchallis** [comment]
You can get a table out of the `blobtools filter` command (in tsv, csv or json format depending on the extension)

```
blobtools filter --table ASSEMBLY_NAME.table.tsv ~/BTK_TUTORIAL/DATASETS/ASSEMBLY_NAME
```

The default output includes the 4 fields that would be shown in the blob plot (equivalent to `--table-fields plot`), but you can include any fields in the table by providing a list to `--table-fields`

```
blobtools filter \
    --table ASSEMBLY_NAME.table.tsv \
    --table-fields gc,length,ncount,bestsumorder_family,bestsumorder_class
    ~/BTK_TUTORIAL/DATASETS/ASSEMBLY_NAME
```

**Comment by pstrope** [comment]
Thank you! That is very useful.

I see this value set for evalue in the example yamls, to be used for BLAST and Diamond. Is this the value used for all the Datasets listed in the website?
`similarity:
  defaults:
    evalue: 1e-25`

**Comment by rjchallis** [comment]
yes, all the blast/diamond results are imported with an evalue cutoff of 1e-25. You can get all the metadata for each assembly through the API with a url like https://blobtoolkit.genomehubs.org/api/v1/dataset/id/CAJPIC01. This has `settings` and `similarity` sections based on the yaml used to run the pipeline.

**Comment by pstrope** [comment]
Thank you Richard. What is the difference between evalue and import_evalue, they have different values?

**Comment by rjchallis** [comment]
the evalue is used when running blast/diamond, import_evalue is used as a cutoff during import. I've been running the blasts at 1e-10 to allow for some weaker hits in the output file while keeping the usual default for import to allow for assessing how much difference the cutoff makes, but haven't actually looked into it yet.

**Comment by rjchallis** [comment]
The database is usually in a dated folder in `similarity > diamond_blastx > path`. I just noticed that the blast n/nt path isn't in the example I sent, but that will be from the same month/year and will match the date of `settings > taxdump`. This system broke for a while when I was running them in dockers with paths mapped to the container filesystem so if there are assemblies you can't find the dates for, let me know and I can try to find the dates for you.

**Comment by pstrope** [comment]
Hi Richard,
In this page, in the first example, you are adding coverage, but why are you adding .json instead of bam? 
https://blobtoolkit.genomehubs.org/blobtools2/blobtools2-tutorials/adding-data-to-a-dataset/adding-coverage/
